docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  haystack-api:
+    image: "deepset/haystack-api-cpu:0.2.0"
```

Comment:
How about naming the image "deepset/haystack-cpu:0.2.0" without the "api"? Is there anything specific for the API in it? We might also re-use it for training / eval ...

Reply:
Yes, that sounds good. The `CMD` for the image is specific to the API, but that can easily be moved to `docker-compose.yml` and `run_docker_gpu.sh`.